src/main/java/lotto/LottoNumbers.java

```diff
@@ -0,0 +1,133 @@
+package lotto;
+
+import lotto.exception.WrongLottoNumberSizeException;
+
+import java.util.ArrayList;
+import java.util.Comparator;
+import java.util.Collections;
+import java.util.List;
+import java.util.HashSet;
+import java.util.Objects;
+import java.util.Set;
+import java.util.stream.Collectors;
+
+public class LottoNumbers {
+    public static final String FIND_ALL_SPACES = "\\s+";
+    public static final String REMOVE_SPACES = "";
+    public static final String LOTTO_NUMBERS_BASE_SEPARATOR = ",";
+    public static final String LOTTO_NUMBERS_DESCRIPTION_OPEN_BRACKET = "[";
+    public static final String LOTTO_NUMBERS_DESCRIPTION_CLOSE_BRACKET = "]";
+    public static final String LOTTO_NUMBERS_DESCRIPTION_SPACE = " ";
+    public static final int LOTTO_SIZE = 6;
+    private static final String WRONG_LOTTO_NUMBER_SIZE_MESSAGE = "입력하신 로또의 개수를 확인 해 주세요.";
+    private static final List<LottoNumber> randomLottoNumberPocket = generateRandomLottoNumberPocket();
+
+    private final Set<LottoNumber> lottoNumbers;
+
+    public LottoNumbers() {
+        this.lottoNumbers = new HashSet<>();
+        generateRandomLottoNumbers();
+    }
+
+    public LottoNumbers(Set<LottoNumber> lottoNumbers) {
+        this.lottoNumbers = lottoNumbers;
+    }
+
+    public LottoNumbers(String lottoNumbersString) {
+        this(lottoNumbersString, LOTTO_NUMBERS_BASE_SEPARATOR);
+    }
+
+    public LottoNumbers(String lottoNumbersString, String separator) {
+        this.lottoNumbers = new HashSet<>();
+        generateLottoNumbersFromString(lottoNumbersString, separator);
+    }
+
+    private static List<LottoNumber> generateRandomLottoNumberPocket() {
+        List<LottoNumber> resultPocket = new ArrayList<>();
+        for (int number = LottoNumber.MIN_BOUND; number <= LottoNumber.MAX_BOUND; number++) {
+            resultPocket.add(new LottoNumber(number));
+        }
+
+        return resultPocket;
+    }
+
+    private void generateRandomLottoNumbers() {
+        Collections.shuffle(randomLottoNumberPocket);
+        for (int index = 0; index < LOTTO_SIZE; index++) {
+            LottoNumber randomNumber = randomLottoNumberPocket.get(index);
+            lottoNumbers.add(randomNumber);
+        }
+    }
```

Comment:
```suggestion
    private Set<LottoNumber> generateRandomLottoNumbers() {
        Collections.shuffle(randomLottoNumberPocket);
        
        return new HashSet<>(randomLottoNumberPocket.subList(0, LOTTO_SIZE);
    }
```
subList를 통해 간결하게 표현할 수 있을거 같아요

Reply:
> subList를 통해 간결하게 표현할 수 있을거 같아요

와우! 감사합니다!